### DISCUSSION TITLE
E-Mail alias generator with self hosted on android

### DISCUSSION THREAD

**Discussion by miranyx** [bitwarden/clients#11633] [Password Manager]
### Code Contribution Proposal

Hey, I would like to implement email generation with self-hosted email aliasing services like in the web extension. My implementation would focus on the new Android client.
## Feature Request
https://community.bitwarden.com/t/custom-api-url-for-self-hosted-email-alias-services-mobile-clients/62057

**Comment by dwbit** [comment]
Hey @benni347 can you confirm if you're looking to contribute code, or to pass along the feature request?

**Reply by miranyx** [reply]
Hey @dwbit I am looking at contributing the code.

**Reply by dwbit** [reply]
Thanks for confirming, I'll add the proposal tag and have the team review.
